lib/src/mock_navigator.dart

```diff
@@ -1,6 +1,27 @@
 import 'package:flutter/material.dart';
 import 'package:mocktail/mocktail.dart';
 
+class _MockMaterialPageRoute extends MaterialPageRoute<void> {
+  _MockMaterialPageRoute({required super.builder});
+
+  void hackOverlays() {
+    for (var i = 0; i < overlayEntries.length; i++) {
```

Comment:
Is there a way to test this method specifically? I feel it's now accidentally covered

Reply:
I feel we don't need any additional tests, since before the change, no test were passing in flutter 3.13